src/GraphQL.SystemTextJson/ObjectDictionaryConverter.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace GraphQL.SystemTextJson
+{
+    /// <summary>
+    /// A custom JsonConverter for converting into a dictionary of objects of their real underlying type.
+    /// </summary>
+    /// <remarks>
+    /// With thanks to @pekkah who shared this from tanka-graphql.
+    /// </remarks>
+    public class ObjectDictionaryConverter : JsonConverter<Dictionary<string, object>>
+    {
+        private readonly JsonSerializerOptions _internalOptions = new JsonSerializerOptions()
```

Comment:
```suggestion
        private readonly JsonSerializerOptions _internalOptions = new JsonSerializerOptions
```

Reply:
This was removed when I removed the Write methods. No longer relevant.